Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Demonic Clothes

## Changelog:
- Demons use darker colors for their clothes.


## Tests:


# How to test:
There are 2 ways:
- Check what Kalimdor satyrs wear.
- Check clothes in the Portrait Editor
  - Press the `~`.
  - Open the Portrait Editor
  - Select `manari_ethnicity` as Ethnicity
  - Find `clothes` gene
  - Find `demonic` sub groups at the end
  - Click Randomize DNA to get new variation
  - Open 3D View to see the whole model

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132381961-af329682-0fa9-4fd6-ae0b-5202f6feb99f.png)
![image](https://user-images.githubusercontent.com/46384992/132382570-5ce3f071-8836-4be7-9e8d-1295866c613a.png)

</details>

Give feedback. Do you like colors? Do you want to see some other color variation?
